### DISCUSSION TITLE
Receiving Alerts via Email for Activated Rules in Playbook

### DISCUSSION THREAD

**Discussion by MAALAOUI-M** [Security-Onion-Solutions/securityonion#11519] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

64

### Storage for /

1 TB

### Storage for /nsm

1 TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am currently working on configuring Security Onion to send alerts via email for activated rules in the playbook. I have successfully set up Suricata to send alerts to my email address with elastalerts, which has been working well. However, I am facing an issue with receiving alerts specifically for the activated rules within the playbook.

I would like to receive alerts generated from playbook when its associated rules are triggered.

I would greatly appreciate any guidance or assistance on properly configuring Security Onion to ensure that alerts from playbook-activated rules are sent to my email. If there are specific configurations or additional steps that I may be missing, please provide detailed instructions to help me resolve this issue.

Thank you for your help !

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
Playbook uses elastalert, so to have a play send you an email you would need to configure elastalert for that rule like below
https://docs.securityonion.net/en/2.4/elastalert.html?highlight=email#email-internal

**Reply by jaypfe** [reply]
You can edit the elastalert rule created by the playbook to send an email by doing ^ above

You can also build a purpose-built emailer elastalert rule. I have that implemented on my 2.3 deployment but not yet on my 2.4 deployment. But basically, point it at the playbook index, match for any, then configure your email settings as exampled in the docs.
That will send an email of any alert that gets fired. You may want to tune it show only crits, highs etc get sent, but thats up to you.
